MapboxCoreNavigationTests/NavigationServiceTests.swift

```diff
@@ -256,7 +280,10 @@ class NavigationServiceTests: XCTestCase {
 
         let willRerouteNotificationExpectation = expectation(forNotification: .routeControllerWillReroute, object: router) { (notification) -> Bool in
             let fromLocation = notification.userInfo![RouteController.NotificationUserInfoKey.locationKey] as? CLLocation
-            return fromLocation == testLocation
+
+            XCTAssertTrue(fromLocation == testLocation)
```

Comment:
I've added `XCTAssertTrue` instead for plain return value to see where test fails more easily. Previously it wasn't really obvious since in case if false is returned expectation would fail with timeout.